docs/Makefile

```diff
@@ -29,39 +34,49 @@ prepare:
 managed: prepare attributes apidocs examples bundles
 
 attributes:
-	mkdir -p "${managed_partials}"
+	mkdir -p "${java_managed_partials}"
+	bin/version.sh | xargs -0  printf ":kalix-jvm-sdk-version: %s" \
+		> "${java_managed_partials}/attributes.adoc"
+	echo ":java-version: 11" \
+		>> "${java_managed_partials}/attributes.adoc"
+	echo ":java_minimum_sdk_version: 0.7.0" \
+		>> "${java_managed_partials}/attributes.adoc"
+	echo ":console: https://console.kalix.io/" \
+		>> "${java_managed_partials}/attributes.adoc"
+	mkdir -p "${spring_managed_partials}"
 	bin/version.sh | xargs -0  printf ":kalix-jvm-sdk-version: %s" \
-		> "${managed_partials}/attributes.adoc"
+		> "${spring_managed_partials}/attributes.adoc"
 	echo ":java-version: 11" \
-		>> "${managed_partials}/attributes.adoc"
+		>> "${spring_managed_partials}/attributes.adoc"
 	echo ":java_minimum_sdk_version: 0.7.0" \
-		>> "${managed_partials}/attributes.adoc"
+		>> "${spring_managed_partials}/attributes.adoc"
 	echo ":console: https://console.kalix.io/" \
-		>> "${managed_partials}/attributes.adoc"
+		>> "${spring_managed_partials}/attributes.adoc"
```

Comment:
Could we maybe replace this with `rsync -a ${java_managed_partials}/ ${spring_managed_partials}`? meaning, just do a copy of the java folder and contents since I'm not sure there will ever be differences..

Reply:
very good point